test/Unit/WrapperRunner/WrapperRunnerTest.php

```diff
@@ -607,7 +607,7 @@ public function testFunctionalParallelization(): void
         $this->bareOptions['--functional'] = true;
 
         $runnerResult = $this->runRunner();
-        self::assertStringContainsString('.......', $runnerResult->output);
+        self::assertStringContainsString('.', $runnerResult->output);
```

Comment:
All the mutants failed in https://github.com/paratestphp/paratest/actions/runs/7444310647/job/20256902107?pr=828 scare me: can we make this assertion a bit more exact?

Reply:
Done, assert is now more exact